assets/css/components.css

```diff
@@ -275,4 +275,29 @@
   .info-box {
     @apply p-4 bg-gray-100 text-sm text-gray-500 font-medium rounded-lg;
   }
+
+  /* Cards */
+  .card-item {
+    @apply cursor-pointer;
+  }
+
+  .card-item:not(.selected) .card-item--logo {
```

Comment:
Nitpick: if using BEM it should be `cart-item__logo` (sub-element) and `cart-item--selected` (modifier modifier). We don't have that many classes and I don't think they consistently follow BEM, so just doing `cart-item-logo` also works, I just wouldn't use `--` for sub-element to avoid confusion :D